app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardBody.js

```diff
@@ -35,27 +35,32 @@ class MappingWizardBody extends React.Component {
           {
             title: __('General'),
             render: () => <MappingWizardGeneralStep />,
-            onClick: () => console.log('on step 1 click')
+            onClick: () => console.log('on step 1 click'),
+            disabled: !this.props.mappingWizardGeneralStep.values
           },
           {
             title: __('Clusters'),
             render: () => this.mappingWizardClustersStepContainer,
-            onClick: () => console.log('on step 2 click')
+            onClick: () => console.log('on step 2 click'),
+            disabled: !this.props.mappingWizardClustersStep.values
           },
           {
             title: __('Datastores'),
             render: () => this.mappingWizardDatastoresStepContainer,
-            onClick: () => console.log('on step 3 click')
+            onClick: () => console.log('on step 3 click'),
+            disabled: !this.props.mappingWizardDatastoresStep.values
```

Comment:
Actually, I think this `disabled` property should be called `unopened` or `uninitialized` or `new` or something. Or it could even be called `visited` and be the reverse value.  I think it's fine to use the code you have, but let's call it what it is, `disabled` implies it's using the validation.